benchmarks/templates/benchmarks/faq.html

```diff
@@ -0,0 +1,101 @@
+{% extends "benchmarks/components/app-view.html" %}
+{% load static %}
+
+{% block banner %}
+<h1>Frequently Asked Questions</h1>
+<p>Explore the sections below for answers to the Brain-Score communities common queries.</p>
+{% endblock %}
+
+{% block info_section %}
+  {% include "benchmarks/tutorials/tutorial-info-section.html" %}
+{% endblock %}
+
+{% block content %}
+
+
+<!-- Each Div is a section itself:-->
+<div class="faq_box leaderboard-table-component">
+    <button class="toggle-button is-size-3-mobile" aria-expanded="false">
+      <span class="benefits_heading"><h3 class="benefits_heading is-size-3-mobile">What is Brain-Score? </h3></span>
+      <span class="icon is-small"><i class="fas fa-caret-down icon-gradient"></i></span>
+    </button>
+    <div class="content is-hidden">
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            1) What is the purpose of Brain-Score?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            The Brain-Score platform aims to yield strong computational models of the brain and mind. We enable researchers to quickly get a sense of the alignment of their model(s) to currently dozens of neural and behavioral measurements, and provide these models to experimentalists to prototype new experiments and make sense of biological data.
+        </p>
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            2) What is a benchmark?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            A benchmark is composed of primate neural and/or behavioral data and metric. When a model is run on a benchmark, the metric outputs a score of how well the data matches the model.
+        </p>
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            3) What is a metric?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            A metric scores how similar two sets of data are. Typically these two sets are model and primate measurements, but metrics are agnostic of the data source and can also be used to compare two primate measurements (e.g. for ceiling estimates).
+        </p>
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            4) What is a model?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            Specifically, "model" here refers to any computational model that implements the BrainModel (for Brain-Score Vision) or ArtificialSubject (for Brain-Score Language) API, which defines the interface that allows a conformant model to run on Brain-Score benchmarks. 
```

Comment:
```suggestion
            Specifically, "model" here refers to any computational model that implements the BrainModel (for Brain-Score Vision) or ArtificialSubject (for Brain-Score Language) API, which defines the interface that allows a conformant model to run on Brain-Score benchmarks. 
            Note that Brain-Score is agnostic to the exact model family, e.g. this could be an artificial neural network or a hand-crafted predictor.
            As long as the model makes predictions that adhere to the API, it can be tested.
```